SU2_CFD/src/numerics/elasticity/nonlinear_models.cpp

```diff
@@ -109,10 +109,18 @@ void CFEM_NeoHookean_Comp::Compute_Stress_Tensor(CElement *element, const CConfi
     Lambda_J = Lambda/J_F;
   }
 
+ /*--- Thermal stress parameters ---*/
+  su2double alpha = config->GetThermal_Expansion_Coeff(); // Coefficient of thermal expansion
+  su2double delta_T = element->GetTemperature() - config->GetTemperature_Ref(); // Temperature difference
```

Comment:
The FEA solver has a way to use different materials in different parts of the structure, so I introduced one alpha per material.
The reference temperature you use here is used by some solver for non dimensionalization, so it was better to use a dedicated temperature.
I also added this stress contribution to all material models.